include/rootston/osk.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ROOTSTON_OSK_H
+#define ROOTSTON_OSK_H
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <assert.h>
+
+#include <wayland-server-core.h>
+
+#include "rootston/desktop.h"
+
+struct roots_input_panel {
+	struct roots_desktop *desktop; // FIXME - remove when views come
```

Comment:
If you expect this TODO to get merged, can you elaborate a bit? If you will resolve prior to merge no worries